### PR TITLE
Master bump version to 4.5.0

### PR DESCRIPTION
**Pull request by obnoxxx**
Now that `release-4.4` is created, proceed with master.

### CODE REVIEW

**Comment by obnoxxx** [comment]
Not sure why it keeps failing. `make gen-latest-csv` keeps generating a diff...

File: deploy/olm-catalog/ocs-operator/4.5.0/ocs-operator.v4.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1617 @@
...
+  - block storage
+  - shared filesystem
+  - object storage
+  links:
+  - name: Source Code
+    url: https://github.com/openshift/ocs-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat Support
+  maturity: alpha
```
Comment (umangachapagain) [inline deploy/olm-catalog/ocs-operator/4.5.0/ocs-operator.v4.5.0.clusterserviceversion.yaml]: Do we still want to keep this as `alpha` ? (it is `alpha` for 4.2, 4.3 and 4.4)

Comment (obnoxxx) [reply deploy/olm-catalog/ocs-operator/4.5.0/ocs-operator.v4.5.0.clusterserviceversion.yaml]: I think alpha for the next version (in master is OK).
But we should adapt the version status in the release branches, I guess.
Then again the CSV is changed for the final D/S product build anyway, so no need to be too picky in the upstream repo at this point?

Comment (umangachapagain) [reply deploy/olm-catalog/ocs-operator/4.5.0/ocs-operator.v4.5.0.clusterserviceversion.yaml]: Makes sense. Need to remember for D/S builds.

**Comment by obnoxxx** [comment]
```
error: error creating buildah builder: Error reading signatures: Error downloading signatures for sha256:36e7f9ef80fab4168014c2c00d491ed0ef19a2c90854b6701cb69df145401153 in registry.svc.ci.openshift.org/ci/clonerefs: received unexpected HTTP status: 504 Gateway Time-out
```

**Comment by obnoxxx** [comment]
Now again this:

```
...
MD5: 42f1e44f9ca13e091544f609f6f88071
uncommitted CSV changes. run 'make gen-latest-csv' and commit results. 
```

Why would it pass locally for me but not in the CI?

**Comment by obnoxxx** [comment]
Now I could reproduce locally: I had to make `git clean`, then it failed `make ocs-operator-ci`.

**Pull request merged**